index.yaml

```diff
@@ -651,28 +693,28 @@ entries:
       catalog.cattle.io/namespace: longhorn-system
       catalog.cattle.io/release-name: longhorn-crd
     apiVersion: v1
-    created: "2021-07-08T20:18:13.68868+08:00"
+    created: "2021-09-01T01:22:38.021539971+08:00"
     description: Installs the CRDs for longhorn.
-    digest: a0bcc369ba8daa495df155bb459d826f711af13a4d82974f195bd0e16508a385
+    digest: 16484e129b9c24fd677dcece4b1ce82a9222e9d665be1076ddbf811a3499d837
     name: longhorn-crd
     type: application
     urls:
-    - assets/longhorn/longhorn-crd-1.1.200+up1.1.2.tgz
-    version: 1.1.200+up1.1.2
+    - assets/longhorn/longhorn-crd-1.2.0.tgz
+    version: 1.2.0
```

Comment:
I think we need to add a new entry rather than modifying the existing one.

Reply:
see my previous comment.